app/Http/Controllers/Admin/AdminUserController.php

```diff
@@ -36,4 +43,91 @@ public function list(): JsonResource
+                    new OA\Property(property: 'email', type: 'string', example: 'jan@jan.pl'),
+                    new OA\Property(property: 'hasEmailVerified', type: 'bool', example: false),
+                    new OA\Property(property: 'password', type: 'string', example: 'password'),
+                    new OA\Property(property: 'passwordExpireAt', type: 'datetime', example: '2023-05-10T20:46:43Z'),
+                    new OA\Property(property: 'hasGlobalAdmin', type: 'bool', example: false),
+                ]
+            )
+        ),
+        tags: ['Admin'],
+        responses: [
+            new OA\Response(response: 201, description: 'Created'),
+            new OA\Response(response: 400, description: 'Bad Request'),
+            new OA\Response(response: 401, description: 'Unauthorised'),
+        ]
+    )] /** @throws ApiException|Throwable */
+    public function post(Request $request, CreateUserService $service): JsonResponse
+    {
+        $data = $request->validate(User::getInsertValidator([
+            'name',
+            'email',
+            'has_email_verified',
+            'password',
+            'password_expire_at',
+            'has_global_admin',
+        ]));
+
+        $result = $service->handle($data);
+
+        return new JsonResponse(data: ['data' => ['id' => $result]], status: 201);
+    }
+
+    #[OA\Patch(
+        path: '/api/v1/admin/user/{id}',
+        description: 'Permissions: ' . Permission::globalAdmin->name,
+        summary: 'Update user',
+        requestBody: new OA\RequestBody(
+            content: new OA\JsonContent(
+                properties: [
+                    new OA\Property(property: 'name', type: 'string', example: 'Jan'),
+                    new OA\Property(property: 'email', type: 'string', example: 'jan@jan.pl'),
+                    new OA\Property(property: 'hasEmailVerified', type: 'bool', example: false),
+                    new OA\Property(property: 'password', type: 'string', example: 'password'),
+                    new OA\Property(property: 'passwordExpireAt', type: 'datetime', example: '2023-05-10T20:46:43Z'),
+                    new OA\Property(property: 'hasGlobalAdmin', type: 'bool', example: false),
+                ]
+            )
+        ),
+        tags: ['Admin'],
+        parameters: [
+            new OA\Parameter(
+                name: 'id',
+                description: 'Resource id',
+                in: 'path',
+                required: true,
+                schema: new Schema(type: 'string', format: 'uuid', example: 'UUID'),
+            )],
+        responses: [
+            new OA\Response(response: 200, description: 'Updated'),
+            new OA\Response(response: 400, description: 'Bad Request'),
+            new OA\Response(response: 401, description: 'Unauthorised'),
+        ]
+    )] /** @throws ApiException|Throwable */
+    public function patch(User $user, Request $request, UpdateUserService $service): JsonResponse
```

Comment:
w tej sytuacji użytkownik powinien się z automatu zbindować, i ten kod w AppServiceProvider.php nic nie zmienia

Reply:
ok, a w przypadku gdy podam jakiś uuid z dupy to sypnie 400? bo w sumie nie sprawdzałem

Reply:
nom